lists/CD-MJDH

```diff
@@ -0,0 +1,48 @@
+{
+  "name": {
+    "en": "Ministry of Justice and Human Rights",
+    "local": "Ministère de la Justice et Droits Humains"
+  },
+  "url": "https://journalofficiel.cd/accueil",
+  "description": {
+    "en": "The Official Journal of the Democratic Republic of the Congo is the government gazette of the country. It publishes major legal official information including a bulletin of acts of companies, associations and political parties registered with the Ministry of Justice and Human Rights. This is referred to as the second part (Deuxieme parties) and is published on the 1st and on the 15th of each month, Organisations apply to the Ministry of Justice and Human Rights (Le Ministre de la Justice et Droits Humain) and are issued with a certificate of registration."
```

Comment:
It might be better to rework this to start with:  "NGOs and associations apply to the Ministry of Justice and Human Rights (Le Ministre de la Justice et Droits Humain) and are issued with a certificate of registration...." Then lead on to mention the gazette?